internal/notification/smtp_notifier.go

```diff
@@ -98,17 +98,18 @@ func (n *SMTPNotifier) startTLS() error {
 		return nil
 	}
 
-	ok, _ := n.client.Extension("STARTTLS")
-	if ok {
-		log.Debugf("Notifier SMTP server supports STARTTLS (disableVerifyCert: %t, ServerName: %s), attempting", n.tlsConfig.InsecureSkipVerify, n.tlsConfig.ServerName)
+	if n.disableRequireTLS {
+		log.Warn("Notifier SMTP server does not support STARTTLS and SMTP configuration is set to disable the TLS requirement (only useful for unauthenticated emails over plain text)")
+	}
```

Comment:
**DO NOT APPLY THIS SUGGESTION.** This should be under the case false if we want to preserve current functionality I think, additionally this will throw errors when we shouldn't we need another case. Something like this:

```suggestion
        case false:
        	switch n.disableRequireTLS {
		case true:
			log.Warn("Notifier SMTP server does not support STARTTLS and SMTP configuration is set to disable the TLS requirement (only useful for unauthenticated emails over plain text)")
		case false:
			return errors.New("Notifier SMTP server does not support TLS and it is required by default (see documentation if you want to disable this highly recommended requirement)")
		}
	}
```